templates/repo/view_file.tmpl

```diff
@@ -55,6 +55,11 @@
 						<video controls src="{{EscapePound $.RawFileLink}}">
 							<strong>{{.i18n.Tr "repo.video_not_supported_in_browser"}}</strong>
 						</video>
+						<img src="{{EscapePound $.RawFileLink}}">
```

Comment:
Why image tag? This would create a video tag, followed immediately by image tag

Reply:
Oops thanks, copy paste error